data/json/snippets/e_logs.json

```diff
@@ -123,6 +123,86 @@
+      },
+      {
+        "id": "local_files_simple_49",
+        "text": "recording_april_20_03:50.mp3\n\nThe background of this recording is underscored by a bazaar swirling noise and muffled cracks emanating through the audio, causing a faint and inexplicable tingling sensation to prickle at the roots of your teeth.  \"Dear Diary,\" mumbles a child’s voice.  \"It’s been a few hours since all the weirdness started, and there are still noises outside, so I guess it isn't over.  We’re still hiding in the armory building, and…\"\n\n\"Hey, kid,\" interjects a woman’s horse voice.  \"We need to save that thing’s charge, so, mind not draining it all.\"\n\nMuttering a reluctant apology, a light thwack and swish jostle the audio as some manner of cover is lowered over the recording device and the child slides it to the floor, the feed still unterminated.  Moments pass, during which only the disquieting sounds of a world in unnatural flux can be heard, accompanied by the clinking of various small metal objects being toyed with next to the microphone.  Eventually, the uncomfortable silence is broken as the woman clears her voice.\n\n\"Listen… I’m sorry.  Honestly, I don’t really care about it too much: it’s just that… I’m just as scared as you are.\"\n\n\"I thought soldiers didn’t get scared.\"\n\nAny form of reply is cut off as a frantic banging suddenly batters into the soundscape: with a man’s muffled shrieks for help audible as the child, alarm raising their voice by several octaves, cries, \"That’s dad!  We've got to open the door!\"\n\nA scramble ensues, eventually culminating in the child’s yelling protests as they’re evidently restrained.\n\n\"Listen to me,\" the woman’s voice barks.  \"Kid, you can’t open that door.  The thing out there ain’t your dad.\"\n\n\"What’re you talking about?\" the youth sobs.  The soldier’s cadence softens with no small measure of sadness as she audibly pulls the struggling child to a sitting position by the microphone.\n\n\"No one knows we’re in here,\" she replies.  \"Nobody would be begging to be let in.  Plus, no one could have lasted this long outside.  Just, cover your ears.  It’ll stop once it realizes it’s not fooling anyone.\"\n\nAs the screams and pleas grow increasingly distorted, eventually verging on a blood-curdling wail, the youth sniffles, \"Then, what is it?\"\n\nLong moments pass, during which the child’s weeping grows muted as they’re pulled into an evident embrace.  \"Not a person,\" the woman eventually whispers.  \"Not a person.\""
```

Comment:
```suggestion
        "text": "recording_april_20_03:50.mp3\n\nThe background of this recording is underscored by a bazaar of swirling noise and muffled cracks emanating through the audio, causing a faint and inexplicable tingling sensation to prickle at the roots of your teeth.  \"Dear Diary,\" mumbles a child’s voice.  \"It’s been a few hours since all the weirdness started, and there are still noises outside, so I guess it isn't over.  We’re still hiding in the armory building, and…\"\n\n\"Hey, kid,\" interjects a woman’s horse voice.  \"We need to save that thing’s charge, so, mind not draining it all.\"\n\nMuttering a reluctant apology, a light thwack and swish jostle the audio as some manner of cover is lowered over the recording device and the child slides it to the floor, the feed still unterminated.  Moments pass, during which only the disquieting sounds of a world in unnatural flux can be heard, accompanied by the clinking of various small metal objects being toyed with next to the microphone.  Eventually, the uncomfortable silence is broken as the woman clears her voice.\n\n\"Listen… I’m sorry.  Honestly, I don’t really care about it too much: it’s just that… I’m just as scared as you are.\"\n\n\"I thought soldiers didn’t get scared.\"\n\nAny form of reply is cut off as a frantic banging suddenly batters into the soundscape: with a man’s muffled shrieks for help audible as the child, alarm raising their voice by several octaves, cries, \"That’s dad!  We've got to open the door!\"\n\nA scramble ensues, eventually culminating in the child’s yelling protests as they’re evidently restrained.\n\n\"Listen to me,\" the woman’s voice barks.  \"Kid, you can’t open that door.  The thing out there ain’t your dad.\"\n\n\"What’re you talking about?\" the youth sobs.  The soldier’s cadence softens with no small measure of sadness as she audibly pulls the struggling child to a sitting position by the microphone.\n\n\"No one knows we’re in here,\" she replies.  \"Nobody would be begging to be let in.  Plus, no one could have lasted this long outside.  Just, cover your ears.  It’ll stop once it realizes it’s not fooling anyone.\"\n\nAs the screams and pleas grow increasingly distorted, eventually verging on a blood-curdling wail, the youth sniffles, \"Then, what is it?\"\n\nLong moments pass, during which the child’s weeping grows muted as they’re pulled into an evident embrace.  \"Not a person,\" the woman eventually whispers.  \"Definitely not a person.\""
```